documentation/ipi-install/modules/ipi-install-upstream-adding-a-worker-node-to-an-existing-cluster.adoc

```diff
@@ -0,0 +1,182 @@
+----
+
+. Ensure the `PROVISIONING STATUS` is `ready` before provisioning the bare metal node.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$  oc -n openshift-machine-api get bmn openshift-worker-<num>
+----
++
+Where `<num>` is the worker node number.
++
+[source,bash]
+----
+NAME                 STATUS   PROVISIONING STATUS   CONSUMER   BMC                 HARDWARE PROFILE   ONLINE   ERROR
+openshift-worker-<num>   OK       ready                            ipmi://<out-of-band-ip>   unknown            true
+----
+
+. Get a count of the number of worker nodes.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ oc get nodes
+----
+
+. Get the machine set.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ oc get machinesets -n openshift-machine-api
+----
+
+. Increase the number of worker nodes by 1.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ oc scale --replicas=<num> machineset <machineset> -n openshift-machine-api
+----
++
+Replace `<num>` with the new number of worker nodes. Replace `<machineset>` with the name of the machine set from the previous step.
+
+. Check the status of the bare metal node.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ oc -n openshift-machine-api get bmh openshift-worker-<num>
+----
++
+Where `<num>` is the worker node number. The status changes from `ready` to `provisioning`.
++
+[source,bash]
+----
+NAME                 STATUS   PROVISIONING STATUS   CONSUMER                  BMC                 HARDWARE PROFILE   ONLINE   ERROR
+openshift-worker-<num>   OK       provisioning          openshift-worker-<num>-65tjz   ipmi://<out-of-band-ip>   unknown            true
+----
++
+The `provisioning` status remains until the {product-title} cluster provisions the node. This may take 30 minutes or more. Once complete, the status will change to `provisioned`.
++
+[source,bash]
+----
+NAME                 STATUS   PROVISIONING STATUS   CONSUMER                  BMC                 HARDWARE PROFILE   ONLINE   ERROR
+openshift-worker-<num>   OK       provisioned           openshift-worker-<num>-65tjz   ipmi://<out-of-band-ip>   unknown            true
+----
+
+. Once provisioned, ensure the bare metal node is ready.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ oc get nodes
+----
++
+[source,bash]
+----
+NAME                                                STATUS   ROLES           AGE     VERSION
+openshift-master-0.openshift.example.com            Ready    master          30h     v1.16.2
+openshift-master-1.openshift.example.com            Ready    master          30h     v1.16.2
+openshift-master-2.openshift.example.com            Ready    master          30h     v1.16.2
+openshift-worker-<num>.openshift.example.com        Ready    worker          3m27s   v1.16.2
+----
++
+You can also check the kubelet.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ ssh openshift-worker-<num>
+----
++
+[source,bash]
+----
+[kni@openshift-worker-<num>]$ journalctl -fu kubelet
```

Comment:
after this, comes 13.3.1 which is in the wrong spot. 

Take a look https://deploy-preview-587--baremetal-deploy.netlify.app/4.6/deployment#ipi-install-upstream-dns-records-for-the-worker

Reply:
What is the right spot?

Reply:
@rlopez133 (ping)